data/compile.py

```diff
@@ -64,39 +64,21 @@ def main():
 
     # TODO: Does it make sense to introduce a type 'sub_building' here?
 
-    for _id, _data in data.items():
-        _data["type_common_name"] = {
-            "root": "Standortübersicht",
-            "site": "Standort",
-            "campus": "Campus",
-            "area": "Gebiet / Gruppe von Gebäuden",
-            "joined_building": "Gebäudekomplex",
-            "building": "Gebäudeteil"
-            if (_data["type"] == "building" and data[_data["parents"][-1]]["type"] == "joined_building")
-            else "Gebäude",
-            "room": _data["usage"]["name"] if "usage" in _data else "Raum",
-            "virtual_room": _data["usage"]["name"] if "usage" in _data else "Raum/Gebäudeteil",
-        }[_data["type"]]
+    logging.info("-- 35 Infer the common_name for every type")
+    structure.infer_type_common_name(data)
 
     logging.info("-- 40 Coordinates")
-    coords.assert_buildings_have_coords(data)
-    coords.assign_coordinates(data)
-    coords.check_coords(data)
+    coords.add_and_check_coords(data)
 
     logging.info("-- 45 Roomfinder maps")
-    maps.assign_roomfinder_maps(data)
-    maps.remove_non_covering_maps(data)
-    maps.assign_default_roomfinder_map(data)
-    maps.build_roomfinder_maps(data)
+    maps.roomfinder_maps(data)
 
     logging.info("-- 46 Overlay maps")
     maps.add_overlay_maps(data)
 
-    logging.info(f"-- 50 convert {images.IMAGE_BASE} to webp")
-    convert_to_webp(images.IMAGE_BASE)
-    logging.info("-- 51 resize and crop the images for different resolutions and formats")
+    logging.info("-- 50 resize and crop the images for different resolutions and formats")
```

Comment:
```suggestion
    logging.info("-- 50 Convert, resize and crop the images for different resolutions and formats")
```
I would add explicitly that this converts the images as well